docs/source/inference.mdx

```diff
@@ -0,0 +1,15 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimum Inference with OpenVINO
+
```

Comment:
It would be great to showcase the following scenarios:
- Out-of-the-box usage
- Static shapes for different pipelines and their parameters (Text Classification, QA, Image Classification)
- GPU usage including precision lowering for faster inference (FP16)

Reply:
I added these examples